content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -205,6 +208,7 @@ providers (Cloud Directory, SAML, social log-in with Google or Facebook etc.) or
 custom providers.
 
 1. Setup environment variables:
```

Comment:
Sorry, missed this one. This word should be a verb - "to set up" - instead of a noun ("a setup"):

```diff
- 1. Setup environment variables:
+ 1. Set up environment variables:
```

Reply:
Same for line 232 please. Thank you!

```diff
- 2. Setup configuration files:
+ 2. Set up configuration files:
```

Reply:
ok I will correct it.

Reply:
Thanks ⚡